<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-FV1AVM16" failures="2" tests="4" name="ErrorValidationTest" time="56.824" errors="0" timestamp="2025-10-24T01:14:04 IST">
  <testcase classname="rahulshettyacademy.tests.errorValidationTest" name="prodErrorValidationTest" time="8.236">
    <skipped/>
  </testcase> <!-- prodErrorValidationTest -->
  <testcase classname="rahulshettyacademy.tests.errorValidationTest" name="prodErrorValidationTest" time="7.874">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rahulshettyacademy.tests.errorValidationTest.prodErrorValidationTest(errorValidationTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- prodErrorValidationTest -->
  <testcase classname="rahulshettyacademy.tests.errorValidationTest" name="zerrorValidationTest" time="4.14">
    <skipped/>
  </testcase> <!-- zerrorValidationTest -->
  <testcase classname="rahulshettyacademy.tests.errorValidationTest" name="zerrorValidationTest" time="4.898">
    <failure type="java.lang.AssertionError" message="expected [Login Successful] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Login Successful] but found [Incorrect email or password.]
at rahulshettyacademy.tests.errorValidationTest.zerrorValidationTest(errorValidationTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- zerrorValidationTest -->
</testsuite> <!-- ErrorValidationTest -->
